github_pr.py

```diff
@@ -108,12 +113,74 @@ def github_list_prs(**args):
     elif 'head' in args and args['head']:
         prs = _load_prs_by_branch(**args)
         _print_prs(prs, **args)
+    elif 'filters' in args and args['filters']:
+        prs = github_filter_prs(**args)
+        _print_prs(prs, **args)
     else:
         prs = repo.get_pulls()
         _print_prs(prs, **args)
     return list_return_obj
 
 
+def github_filter_prs(**args):
+    """
+    Filters prs to return only what is contained in the filters
+    """
+    git_hub = Github(args['token'])
+    repo = git_hub.get_repo(args['repo'])
+    all_prs = [{'pr':repo.get_pull(pr.number), 'issue':repo.get_issue(pr.number)} for pr in repo.get_pulls()]
+    filters = {}
+    for filter_option in args['filters'].split(','):
+        part = filter_option.partition("=")
+        filters[part[0]] = part[2]
+
+    if 'owner' in filters:
+        all_prs = return_specific_owner_prs(all_prs, filters)
+    if 'label' in filters:
+        all_prs = return_specific_labeled_prs(all_prs, filters)
+    if 'status' in filters:
+        all_prs = return_specific_status_prs(all_prs, filters)
+    if 'comment' in filters:
+        all_prs = return_specific_comment_prs(all_prs, filters)
+
+    return all_prs
```

Comment:
It is not just a list of dictionaries with 1 entry. the dictionary looks like

all_prs = [{'pr':pr_obj, 'issue':issue_obj}]

this is necessary because we need information inside the issue obj that is not contained in the pr obj. I think we  want to keep it like this because we will need the issue obj if this is called via a script as a library.